src/main/java/thePackmaster/cards/contentcreatorpack/Rhapsody.java

```diff
@@ -0,0 +1,28 @@
+package thePackmaster.cards.contentcreatorpack;
+
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.powers.bardinspirepack.InspirationPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.applyToSelf;
+import static thePackmaster.util.Wiz.atb;
+
+public class Rhapsody extends AbstractContentCard {
+    public final static String ID = makeID("Rhapsody");
+    public Rhapsody() {
+        super(ID, 2, CardType.SKILL, CardRarity.RARE, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 150;
```

Comment:
That's a lot of inspiration -- 2.5x the effect of a damage or block card, or 3x the effect when upgraded. You don't get increases to secondary effects of the cards, so there are things that Burst/Double Tap/etc. can do that this can't, but this seems really high efficiency when you consider that it also draws 2 cards.

It also seems very high efficiency when you consider that bards 3-cost form power (Splendid Form) only gives 100 Inspiration at the start of each turn.

My gut reaction is that this should be toned down slightly -- don't need to overnerf it, just reduce the inspiration and/or draw slightly. It's fine to have powerful cards, so maybe the thing to do is to reduce the draw to 1. I think it'll still be a very good card (and incredible with Snecko Eye).